tartiflette/directive/directive.py

```diff
@@ -32,7 +33,7 @@ def __init__(self, name: str, schema_name="default"):
 
     def bake(self, schema):
         if not self._implementation:
-            raise Exception("No implementation given")
+            raise MissingImplementation("No implementation given")
```

Comment:
Can you add the directive name in the message ? "No Implementation given for directive < %s >" or something like that.